### DISCUSSION TITLE
matchAgeMinDays for delaying delaying inclusion

### DISCUSSION THREAD

**Discussion by betaorbust** [renovatebot/renovate#15224] [Suggest an Idea]
### What would you like Renovate to be able to do?

I'd like to be able to have a group of dependencies raised in a change, but only including packages that have had some bake time (for security, stability, etc.) so that auto-merge can be used with them. 
I understand `stabilityDays` is available, but it will hold off automerging a PR until _all_ involved packages are X days old, which is less useful in large groups or with larger timeframes when packages/libraries are constantly being updated. 

### If you have any ideas on how this should be implemented, please tell us here.

I think this could be implemented with a `matchAgeMinDays` matcher. It's not maybe a perfect DX because I'm excluding everything and then putting it back in, but it enables this use case of "I want a group of package updates, but only including packages that are older than X days, that can be automerged" and is, I think, a good building block.

```js
{
    "packageRules": [
        {
            "description": "Start by ignoring all minor/patch updates.",
            "matchUpdateTypes": ["minor", "patch"],
            "enabled": false
        },
        {
            "description": "Then opt-in sufficiently aged minor/patch updates as a group.",
            "matchUpdateTypes": ["minor", "patch"],
            "matchAgeMinDays": 5,
            "enabled": true
        },

}

```

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by viceice** [comment]
https://docs.renovatebot.com/configuration-options/#internalchecksfilter ?

**Comment by HonkingGoose** [comment]
I think this issue is also relevant:

- https://github.com/renovatebot/renovate/issues/12914

**Reply by rarkins** [reply]
Yes, let's consolidate discussions into https://github.com/renovatebot/renovate/issues/12914 if `internalChecksFilter` doesn't satisfy. I've converted this one into a Discussion
